scripts/Dockerfile.test

```diff
@@ -7,8 +7,7 @@
 # Operating System, so generated binaries and versions of dependencies may be a
 # little different.
 #
-
-FROM quay.io/fedora/fedora:latest
+FROM quay.ceph.io/ceph-ci/ceph:wip-pkalever-rbd-group-snap-mirror
```

Comment:
Isn't this getting garbage collected? I thought there was a copy somewhere that would stay around longer?

Reply:
Not anytime soon, at least not till the upstream PR is merged.